TODOList.md

```diff
@@ -0,0 +1,14 @@
+# Google PACO Source Code List of TODO
```

Comment:
Certainly. 

We have already started down the road of upgrading the iOS model to the model in the Android and server side. We are transpiling the model classes from Shared/src (Java) using j2objc so as to reduce duplicate code for the experiment objects across the platforms. We have new UI mocks as well that we are working towards (though those will need revision (as we learned in the Android upgrade)).

I will put together a wiki page with the iOS work plan in the next day or so and ping you when it is up.

Oh, one other thing. In order for us to accept code submissions, Google requires any contributors to sign the Contributor License agreement. Please sign it (either individually, or, if you are contributing on part of a company, have someone who can legally sign for your company sign the corporate one). Please let me know when this is done so I can verify it. You can find the CLA here: https://cla.developers.google.com/clas

Best,
Bob

> On Aug 10, 2015, at 4:50 PM, Dmitriy Groschovskiy <notifications@github.com> wrote:
> 
> In TODOList.md:
> 
> > @@ -0,0 +1,14 @@
> > +# Google PACO Source Code List of TODO
> 
> I would like to translate the application for Swift. Can I take part in the development of the next version of the application?
> 
> —
> Reply to this email directly or view it on GitHub.
>